src/main/resources/templates/cluster-overview.ftl

```diff
@@ -150,6 +150,33 @@
             </tbody>
         </table>
     </div>
+    <div id="create-form">
+        <form action="<@spring.url '/topic'/>" method="POST">
+            <button class="btn btn-outline-light" type="submit">
+                Create topic
+            </button>
+            <table class="table table-bordered" style="width: 50%; margin-top: 20px">
+                <tbody>
+                <tr>
+                    <td>Topic name</td>
+                    <td><input type="text" name="name" value="NewTopic" required></td>
+                </tr>
+                <tr>
+                    <td>Partitions number</td>
+                    <td><input type="number" name="partitionsNumber" value="1" required></td>
+                </tr>
+                <tr>
+                    <td>Replication factor</td>
+                    <td><input type="number" name="replicationFactor" value="1" required></td>
+                </tr>
+                <tr>
+                    <td>Create timeout, ms</td>
```

Comment:
Do you think there's value in having a user-configurable creation timeout? I was thinking maybe something sensible, that we can set in `application.yml`.

Reply:
Yes, I agree, It's rare case to set custom creation timeout. Do you think 2000 ms is ok for standard timeout?

Reply:
Looking at the official [admin client configs](https://kafka.apache.org/documentation/#adminclientconfigs), it looks like the default timeout is set by passing the `request.timeout.ms` property, which defaults to 120000. 

So, option 1: we could just omit the timeout in the request, taking whatever the value of the default is. 

Option 2: set the timeout in `application.yml` and have the same default as Kafka, i.e. 2 mins. If the user wants to override it, they can change the config file — either directly, or via a property or env variable if running from Docker. 

Of course, we could also set our own, but 2 seconds might be aggressive, especially if using a managed Kafka service (like Aiven or Instaclustr) where the brokers are not on a local network.

Reply:
Omit timeout, so default value will be used